src/components/ReplyComment/index.js

```diff
@@ -47,6 +47,7 @@ const ReplyCommentId = ({
     updateFeed,
     scrollViewRef,
     createComment,
+    onSaveHeight,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new prop `onSaveHeight` is added to the hook `useReplyComment`. Ensure that this prop is properly defined and used within the hook. If it's optional, make sure there are checks in place to handle its absence.

```diff
-     createComment,
+     onSaveHeight,
      getThisComment
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->